identity/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## 0.1.2 - unreleased
 
+- Clarified deprecation notes on `Keypair` and `PublicKey`
+- Add `impl From<ed25519::PublicKey> for PublicKey` so that `PublicKey::from(ed25519::PublicKey)` works
+- Added `PeerId::to_public_key()`
```

Comment:
Let's focus the changelog entry on these two items please.

Can you combine them into one bullet point such that  it is clear that the PR reference applies to both?

Reply:
Combined/Removed!